core/src/main/java/org/kiwix/kiwixmobile/core/CoreApp.java

```diff
@@ -63,6 +60,8 @@ public static CoreComponent getCoreComponent() {
     return coreComponent;
   }
 
+  public static ActivityComponent getActivityComponent() { return activityComponent; }
```

Comment:
I believe this is unused because there is no set method, if you were to store an ActivityComponent on the Application object you would 100% be causing a memory leak